project/src/main/ui/settings/settings-dialogs.gd

```diff
@@ -13,6 +13,9 @@ signal delete_confirmed
 ## emitted when the player is prompted to delete a save slot and answers 'no' or closes the window
 signal delete_cancelled
 
+## emitted when a dialog becomes visible/invisible
+signal dialog_visibility_changed(value)
```

Comment:
This is worth a comment; its behavior is somewhat surprising; as 'value' could be the dialog